src/ui/screens/onboarding/OnboardingStartSlideDesktop.qml

```diff
@@ -61,11 +62,11 @@ ColumnLayout {
         MZSettingsToggle {
             objectName: "startAtBootToggle"
 
-            checked: MZSettings.startAtBoot
+            checked: startAtBootCheckbox
             onClicked: {
-                MZSettings.startAtBoot = !MZSettings.startAtBoot
+                startAtBootCheckbox = !startAtBootCheckbox
 
-                if (MZSettings.startAtBoot) {
+                if (startAtBootCheckbox) {
```

Comment:
Unrelated to this PR but does it seem to you like we should we be collecting this telemetry when the view is destroyed or when the "Next" button is clicked, instead of here when the checkbox/toggle is clicked? IIUC, if the user disables this checkbox, we'll record a `connectOnStartupDisabled` event, but they'll have to re-enable the toggle before we'll record `connectOnStartupEnabled`.

Reply:
I see reasons for leaving it here, or putting down with the "next" work. I don't feel strongly on this one.
- Move it to "next": This would let us only record it once for this screen.
- Leave it here: It records the user changing their mind (from the default or their previous thought). Also, moving it  to "next" would mean we'd always record at least one change on this screen, even if the user didn't change anything (unless we add a new `didChangeSwitch` boolean, but that feels like possibly overengineering this). Additionally, it's how it currently functions, so it's easier to compare data across client versions if we leave it here.

I'm inclined to leave it here, but could easily be talked into moving it to the "next" logic.